src/ralph/back_office/admin.py

```diff
@@ -1,15 +1,48 @@
 # -*- coding: utf-8 -*-
 from django.utils.translation import ugettext_lazy as _
 
-from ralph.admin import RalphAdmin, register
+from ralph.admin import RalphAdmin, RalphTabularInline, register
+from ralph.admin.views.extra import RalphDetailViewAdmin
 from ralph.back_office.models import BackOfficeAsset, Warehouse
 from ralph.back_office.views import (
     BackOfficeAssetComponents,
-    BackOfficeAssetLicence,
     BackOfficeAssetSoftware
 )
 from ralph.data_importer import resources
 from ralph.lib.permissions.admin import PermissionAdminMixin
+from ralph.licences.models import BaseObjectLicence
+from ralph.supports.models import Support
+
+
+class BackOfficeAssetSupport(RalphDetailViewAdmin):
+    icon = 'bookmark'
+    name = 'bo_asset_support'
+    label = _('Supports')
+    url_name = 'back_office_asset_support'
+
+    class BackOfficeAssetSupportInline(RalphTabularInline):
+        model = Support.base_objects.through
```

Comment:
maybe it'll be better to use `BackOfficeAsset.supports.related.through` here? It's the same table - the only change it that in your example we're going to it from support (touching base object) and in mine there is straightforward way from BO asset to supports.